go.sum

```diff
@@ -2259,6 +2259,8 @@ google.golang.org/grpc v1.40.0/go.mod h1:ogyxbiOoUXAkP+4+xa6PZSE9DZgIHtSpzjDTB9K
 google.golang.org/grpc v1.47.0/go.mod h1:vN9eftEi1UMyUsIF80+uQXhHjbXYbm0uXoFCACuMGWk=
 google.golang.org/grpc v1.55.0 h1:3Oj82/tFSCeUrRTg/5E/7d/W5A1tj6Ky1ABAuZuv5ag=
```

Comment:
Are these expected to still be here?

Reply:
I think so. We have older versions of grpc as well (v1.47.0)

Reply:
Ah, sorry. Forgot to run `go mod tidy`